### PR TITLE
Added link to the real plugin repo

### PR DESCRIPTION
**Pull request by williamhatcher**

### CODE REVIEW

File: plugins.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
 	<h2>Install Plugins</h2>
 	<p>There are many plugins available to use, and installing them is easy. We have a plugin repository where you can find most plugins available for Cuberite. The User's Manual contains instructions on how to install a plugin.</p>
 	<a href="https://book.cuberite.org/#2.5" class="button">Installation Guide</a>
+	<a href="https://plugins.cuberite.org/" class="button">Plugin Repository</a>
 	<a href="https://forum.cuberite.org/forum-2.html" class="button">Download Plugins</a>
```
Comment (bearbin) [inline plugins.html]: The text here doesn't really make sense in connection with the previous line, could you change it?

**Review by bearbin — CHANGES_REQUESTED** [review]

File: plugins.html (hunk 1 above)
Comment (williamhatcher) [reply plugins.html]: Sure.

**Comment by madmaxoft** [comment]
I don't think the plugin repository is anywhere near being usable, so it's still preferable not to link to it just yet.

**Review by bearbin — APPROVED** [review]

**Comment by bearbin** [comment]
I have approved this PR - although the plugin repository is not particularly usable at present, I don't believe there is any harm in linking to it in addition to the forum.

**Comment by madmaxoft** [comment]
Okay then.

**Comment by williamhatcher** [comment]
Yeah, I figured having both links would cover all bases.

**Pull request merged**